runtime/bindings/python/src/pyopenvino/CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ else()
 endif()
 
 target_include_directories(${PROJECT_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/..")
-target_link_libraries(${PROJECT_NAME} PRIVATE openvino::runtime openvino::frontend::manager)
+target_link_libraries(${PROJECT_NAME} PRIVATE openvino::runtime ${OFFLINE_TRANSFORMATIONS_LIB} openvino::frontend::manager)
```

Comment:
why is it needed? should be used only in MO / POT

Reply:
I think it needs hear https://github.com/openvinotoolkit/openvino/blob/master/runtime/bindings/python/src/pyopenvino/core/offline_transformations.cpp#L9